pandamium_datapack/data/pandamium/functions/on_join.mcfunction

```diff
@@ -3,8 +3,11 @@ execute as @s run function pandamium:misc/update_teams
 
 scoreboard players reset @s[scores={gameplay_perms=3..}] home_cooldown
 scoreboard players reset @s[scores={gameplay_perms=3..}] tpa_cooldown
-execute if score @s particles matches 1.. unless score @s gameplay_perms matches 3.. run scoreboard players set @s particles 0
-execute if score @s gameplay_perms matches 3.. run scoreboard players enable @s pose
+
+execute if score @s active_particle matches 1.. unless score @s gameplay_perms matches 3.. run scoreboard players set @s active_particle 0
+
+execute if score @s particles matches 1.. run scoreboard players operation @s active_particle = @s particles
```

Comment:
Is it necessary to have this in `on_join`? It should be run in the normal trigger check function in a few ticks anyways, seems like duplicate commands to me